release/README.md

```diff
@@ -14,3 +14,33 @@ This directory provides a script for releasing new versions of the application.
    export REPO_PREFIX=gcr.io/bank-of-anthos # optional. Modify if you want to push to other repo
    ./make-release.sh
 ```
+
+---
+
+## Minor Version Updates
+
+Occasionally, you may create a pull request includes changes that are incompatible with the latest published container images.
+To resolve this, you need to push a new minor version update along with your code changes:
+
+1. ensure you have an up-to-date, local copy of your pull request branch on your machine
+1. set the NEW_VERSION environment variable to reflect the new minor update (increment Z)
+  `export NEW_VERSION=vX.Y.Z`
+1. run the `./make-release.sh` script. This will push new images, open a new `release/vX.Y.Z` branch, and push it to ``origin``
+1. open a pull request from the new release branch into your original branch
+1. if there are any CI issues on the release pull request and you need to make changes, make sure you keep the release artifacts in sync:
```

Comment:
I see bash commands below and I have a very strong urge to run them.

After reading this step, it's not clear to me that I need to run these commands.

Suggestion:  clean up this sentence

>  if there are any CI issues on the release pull request and you need to make changes, make sure you keep the release artifacts in sync:

"if you make any changes to the pull request, run the following commands to keep the release artifacts in sync:"